rmgpy/data/kinetics/library.py

```diff
@@ -150,7 +150,8 @@ def getLibraryReactions(self):
                  )
             elif entry._longDesc and 'rate rule' in entry._longDesc: #template reaction
                 c = entry._longDesc.split('\n')
-                familyname = c[-1].replace('family: ','')
+                cnew = [i for i in c if 'family: ' in i]
```

Comment:
It is not intuitively obvious from the variable name that 'cnew' contains only strings of family names. Maybe we could rename it to 'family_comments' or something like that.

Reply:
fixed